descqa/apparent_mag_func_test.py

```diff
@@ -97,7 +106,8 @@ def get_validation_data(self, band, observation):
 
         validation_data = dict(zip(data_args['colnames'], data))
         validation_data['label'] = data_args['label']
-
+        if self.skip_label_detail:
+            validation_data['label'] = re.split('\(', validation_data['label'])[0]
```

Comment:
```suggestion
            validation_data['label'] = re.split(r'\(', validation_data['label'])[0]
```

Reply:
I fixed this by using rpartition instead.